src/Symfony/Component/ExpressionLanguage/ParserCache/ParserCacheInterface.php

```diff
@@ -15,6 +15,8 @@
 
 /**
  * @author Adrien Brault <adrien.brault@gmail.com>
+ *
+ * @deprecated since version 3.2, to be removed in 4.0. Use Psr\Cache\CacheItemPoolInterface instead.
```

Comment:
I would add the ``trigger_error`` call in this file too (not an issue, as Symfony will never use this interface in a project anymore, as the new system does not use it)